tests/setup/mod.rs

```diff
@@ -97,22 +97,26 @@ impl NodeHandle {
     }
 
     #[cfg(feature = "nigiri")]
-    pub fn new_with_lsp_setup() -> NodeHandle {
-        nigiri::start();
-
-        // to open multiple channels in the same block, multiple UTXOs are required
-        for _ in 0..10 {
-            nigiri::fund_node(NodeInstance::LspdLnd, 0.5);
-            nigiri::fund_node(NodeInstance::NigiriLnd, 0.5);
-            nigiri::fund_node(NodeInstance::NigiriCln, 0.5);
+    pub fn new_with_lsp_setup(reset: bool) -> NodeHandle {
```

Comment:
Thanks!

This is somewhat more complex than just making `ensure_lspd_running()` public, because of the unfortunate structure of `new_with_lsp_setup` . I will fix this in a follow-up PR.